specs/commandline/inspect.md

```diff
@@ -0,0 +1,271 @@
+    ├──<signedAttributes>
+    ├──<unsignedAttributes>
+    ├──<certificates>
+    └──<payload>
+```
+
+## Outline
+
+```text
+Inspect all signatures with the signed artifact.
+
+Usage:
+    notation inspect [flags] <reference>
+  
+Flags:
+   -h, --help              help for describing the signature
+   -o, --output json       output on command line sets the output to json
+   -p, --password string   password for registry operations (default to $NOTATION_PASSWORD if not specified)
+       --plain-http        registry access via plain HTTP
+   -u, --username string   username for registry operations (default to $NOTATION_USERNAME if not specified)
+```
+
+## Usage
+
+### Display the details of all the listed signatures and its associated certificate properties of the signed container image
+
+
+```text
+notation inspect [flags] <registry>/<repository>@<digest>
+```
+
+## Inspect signatures on the supplied OCI artifact identified by the digest
+
+```shell
+# Prerequisites: Signatures are stored in a registry referencing the signed OCI artifact
+notation inspect localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+```
+
+An example output:
+```shell
+localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+└──application/vnd.cncf.notary.signature
+    ├── sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+      ├──signingAlgorithm: RSASSA-PSS-SHA-256
+      ├──signedAttributes
+            ├──contentType: application/vnd.cncf.notary.payload.v1+json
+            ├──signingScheme: notary.default.x509
+            ├──signingTime: 2022-04-06T07:01:20Z
+            ├──expiry: 2022-10-06T07:01:20Z
+            ├──io.cncf.notary.verificationPlugin: com.example.nv2plugin     //extended attributes to support plugins
+      ├──userDefinedAttributes
+            ├──io.wabbit-networks.buildId: 123                              //user defined payload annotations.
+      ├──unsignedAttributes
+            ├──io.cncf.notary.timestampSignature: <Base64(TimeStampToken)>  //how TSA response (time stamp token) is represented.
+            ├──io.cncf.notary.signingAgent: notation/1.0.0                  //provides the identifier of a client (e.g. Notation) that produced the signature
+      ├──certificates
+            ├──SHA1Fingerprint: 2f1cc5b8455381cdefac83b4bd305b789cc9c16e
+              ├──issuedTo: Microsoft Root Certificate Authority 2010
+              ├──issuedBy: Microsoft Root Certificate Authority 2010
+              ├──expires: Sat Jun 23 22:04:01 2035
+            ├──SHA1Fingerprint: 8BFE3107712B3C886B1C96AAEC89984914DC9B6B
+              ├──issuedTo: Microsoft Code Signing PCA 2010
+              ├──issuedBy: Microsoft Root Certificate Authority 2010
+              ├──expires: Sun Jul 06 20:50:17 2025
+      └──payload                                                           //descriptor of the target artifact manifest that is signed.
```

Comment:
The word `payload` is confusing. Should it be `target artifact`?

Reply:
> What if the `payload` is not parsable? or even the signature is not parsable?
> 
> Users would like to see: "OK. There is one signature failed to parse but I am able to inspect other signatures" instead of just one single error message without other outputs.

Changed to "signed artifact" as per the discussion and alignment in the community meeting as of 01/09/2023